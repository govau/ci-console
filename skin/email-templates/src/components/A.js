import React from "react";
import styled from "styled-components";

const A = styled.a`
  color: ${({ theme }) => theme.linkColor};
`;

export default ({ children, href, className }) => (
  <A href={href} className={className}>
    {children}
  </A>
);
