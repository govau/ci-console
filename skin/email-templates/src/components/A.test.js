import React from "react";
import renderer from "react-test-renderer";

import A from "./A";

test("renders without crashing", () => {
  const tree = renderer.create(
    <A href="https://example.com" className="override">
      content
    </A>
  );
  expect(tree).toMatchSnapshot();
});
