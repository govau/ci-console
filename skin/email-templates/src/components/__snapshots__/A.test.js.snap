// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<a
  className="override "
  href="https://example.com"
>
  content
</a>
`;
